packages/animations/test/fade_scale_transition_test.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: avoid_as
```

Comment:
Why? Please leave a comment per flutter style guide.

Reply:
(we may just want to turn this lint off in the analyzer options)

Reply:
Same in other files.

Reply:
How can we turn this lint off? I tried adding an `analyzer_options.yaml` in `packages/animations/`, but it didn't work.

**Should we edit the `analyzer_options.yaml` file in the project root directory?**

Reply:
It should be fine to turn this off in the project root directory.

Reply:
Done. Thank you!